integration-tests/tests/specs/features/file-sharing.spec.ts

```diff
@@ -58,24 +58,19 @@ test('submit two sequences with one file each', async ({ pageWithGroup, page })
         ],
     );
 
-    let cleanup: () => Promise<void>;
-    try {
-        cleanup = await submissionPage.uploadExternalFiles('raw_reads', {
-            sub1: {
-                'foo.txt': 'Foo',
-            },
-            sub2: {
-                'bar.txt': 'Bar',
-            },
-        });
+    await page.getByRole('heading', { name: 'Extra files' }).scrollIntoViewIfNeeded();
+
+    await submissionPage.uploadExternalFiles('raw_reads', {
+        sub1: {
+            'foo.txt': 'Foo',
+        },
+        sub2: {
+            'bar.txt': 'Bar',
+        },
+    });
```

Comment:
I agree with @fhennig's comment in [Slack](https://loculus.slack.com/archives/C05G172HL6L/p1749167902426519) that cleaning up is a good practice and the code also doesn't seem overly complex to me. If you think that the cleanup should be removed, let's maybe discuss it in a separate PR?